{"cveId": "CVE-2020-15329", "cwe": ["CWE-732", "CWE-732"], "cvss": [{"baseScore": "5.3", "version": "3.1", "Attack Vector": "None", "Attack Complexity": "Low", "Privileges Required": "None", "User Interaction": "None", "Scope": "Unchaged", "Confidentiality Impact": "Low", "Integrity Impact": "None", "Availability Impact": "None"}], "references": ["https://pierrekim.github.io/blog/2020-03-09-zyxel-secumanager-0day-vulnerabilities.html", "https://www.zyxel.com/support/vulnerabilities-of-CloudCNM-SecuManager.shtml"], "description": ["Zyxel CloudCNM SecuManager 3.1.0 and 3.1.1 has weak Data.fs permissions."], "published": "2022-09-29T03:15:13.120", "state": "PUBLIC", "vendorName": ["n/a"], "productName": ["n/a"], "github": {"advisories": [], "commits": [], "pocAdvisorie": null, "repo": null, "info": {}}, "pocList": []}